packages/trader/src/AppV2/Components/ContractDetailsFooter/contract-details-footer.tsx

```diff
@@ -82,7 +82,9 @@ const ContractDetailsFooter = observer(({ contract_info }: ContractInfoProps) =>
                 <Button
                     label={
                         is_valid_to_sell
-                            ? `${cardLabels.CLOSE} @ ${FormatUtils.formatMoney(bid_price || 0)} ${currency}`
+                            ? `${cardLabels.CLOSE} @ ${FormatUtils.formatMoney(bid_price || 0, {
+                                  currency: currency as any, // currency types mismatched between utils and shared
```

Comment:
going forward when we will be fixing any, the currency type in utils or shared which one is the right one? type casting with one of them will it help ?

Reply:
I'd say it should be fixed in utils for `formatMoney` since the weird type is used there, not on our side, we as customers shouldn't face type incompatibilities while utilizing different shared resources like api-types vs utils in this case. The second best approach is definitely type casting, but because the type in utils is specifically weird (you may check in Kate's screenshot above) I used `any` here.